lib/services/database.go

```diff
@@ -585,6 +757,32 @@ func rdsTagsToLabels(tags []*rds.Tag) map[string]string {
 	return labels
 }
 
+// IsAzureMySQLVersionSupported returns true if database supports IAM authentication.
+// Only available for 5.7 and newer.
+func IsAzureMySQLVersionSupported(version armmysql.ServerVersion) bool {
+	switch version {
+	case armmysql.ServerVersionEight0, armmysql.ServerVersionFive7:
+		return true
+	case armmysql.ServerVersionFive6:
+		return false
+	default:
+		log.Warnf("Unknown Azure MySQL server version: %q", version)
+		return false
+	}
+}
+
+// IsAzurePostgresVersionSupported returns true if database supports IAM authentication.
+func IsAzurePostgresVersionSupported(version armpostgresql.ServerVersion) bool {
+	switch version {
+	case armpostgresql.ServerVersionNine5, armpostgresql.ServerVersionNine6, armpostgresql.ServerVersionTen,
+		armpostgresql.ServerVersionTen0, armpostgresql.ServerVersionTen2, armpostgresql.ServerVersionEleven:
+		return true
+	default:
+		log.Warnf("Unknown Azure Postgres server version: %q", version)
+		return false
+	}
+}
```

Comment:
This logic doesn't scale well. For instance what will happen if Azure will start support 8.1 ?  This logic will need to be aligned.

Reply:
True, how do you feel about returning False if it's MySQL 5.6, and otherwise true?